openeo_processes_dask/process_implementations/comparison.py

```diff
@@ -0,0 +1,140 @@
+from typing import Callable, Optional
+
+import dask.array as da
+import numpy as np
+import xarray as xr
+from numpy.typing import ArrayLike
+from xarray.core.duck_array_ops import notnull
+
+__all__ = [
```

Comment:
WHy did the comparison processes suddenly appear in this PR?

Reply:
In the `logic` processes, I want to add here, I really need some of the `comparison` processes - so, I think the `comparsion` merge wants to go first  and then the `logic` processes can make use of it. I now only added the required processes here, so that I can check if the data includes NaNs, or null values in the same way it was done in `comparison`. I will update this, if there will be changes to the `is_valid` process.